modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/exportimport/staged/model/repository/LayoutPageTemplateCollectionStagedModelRepository.java

```diff
@@ -49,7 +49,7 @@ public LayoutPageTemplateCollection addStagedModel(
 
 		return _layoutPageTemplateCollectionLocalService.
 			addLayoutPageTemplateCollection(
-				userId, serviceContext.getScopeGroupId(),
+				null, userId, serviceContext.getScopeGroupId(),
```

Comment:
I think this should be layoutPageTemplateCollection.getExternalReferenceCode()

Reply:
see https://github.com/liferay-echo/liferay-portal/pull/15607